.github/workflows/ci.yml

```diff
@@ -22,7 +22,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        rust_version: [1.57.0, stable, nightly]
+        rust_version: ['1.60.0', stable, nightly]
```

Comment:
Not sure if bumping straight to 1.60 worth it? We might want to keep msrv only for linux/bsd though, since there's no such thing really as a system rust installation. But that should be addressed in a separate PR.

And in general every change to the CI should be done in a separate patch.

Reply:
> Not sure if bumping straight to 1.60 worth it? We might want to keep msrv only for linux/bsd though, since there's no such thing really as a system rust installation. But that should be addressed in a separate PR.

Yeah, unsure here too, will mull this over a bit. `objc2` requires 1.60 because of changes to `Cargo.toml`, so unsure how that affects other platforms that don't use `objc2`.

> And in general every change to the CI should be done in a separate patch.

I think it makes sense to do them together, so that you easily know which PR needs a specific MSRV, but I can see why it makes sense to do them separately given that all maintainers might not review my iOS/macOS PRs.

Reply:
I moved the bump to https://github.com/rust-windowing/winit/pull/2453.